src/jsonapi-datastore.js

```diff
@@ -11,6 +11,7 @@
     this.id = id;
     this._type = type;
     this._attributes = [];
+    this._associations = [];
```

Comment:
Yeah, I agree `_inverse_associations` is not sexy, but I'm afraid `_associations` might be confusing.